hack/envtest/_matrix/v1.35.0-alpha.3.yaml

```diff
@@ -0,0 +1,2 @@
+go: 1.24
```

Comment:
Should be go 1.25 (https://github.com/kubernetes/kubernetes/blob/v1.35.0-alpha.3/.go-version but without patch version please)